Mathlib/Data/Nat/Count.lean

```diff
@@ -5,7 +5,7 @@ Authors: Yaël Dillies, Vladimir Goryachev, Kyle Miller, Kim Morrison, Eric Rodr
 -/
 import Mathlib.Algebra.Group.Nat.Range
 import Mathlib.Algebra.Order.Ring.Nat
```

Comment:
Any chance you can get reduce this import too?

Reply:
Possibly, but I'll leave that for a next PR